dagmc_stats/dagmc_stats.py

```diff
@@ -297,3 +297,157 @@ def get_coarseness(my_core, meshset, entity_ranges, geom_dim):
         coarseness.append(len(surf_area)/sum(surf_area))
 
     return coarseness
+
+
+def get_tri_vert_data(my_core, native_ranges):
+    """
+    Build a numpy strcutured array to store triangle and vertex related data in the form of
```

Comment:
PEP8: The docstring should start on the same line as the `"""`. And line length of comments and doctrings should be no longer than 72 characters.